dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileListener.java

```diff
@@ -0,0 +1,22 @@
+package com.dotmarketing.portlets.fileassets.business;
+
+/**
+ * Listener to handle a File events
+ * @author jsanca
+ */
+public interface FileListener {
+
+    /**
+     * Gets the identifier, by default the class name
+     * @return String
+     */
+    default String getId () {
+        return this.getClass().getName();
+    }
+
+    /**
+     * When a file that is being modified this event is triggered, match the listener criteria,
+     * @param fileEvent {@link FileEvent}
+     */
+    default void fileModify(FileEvent fileEvent) {}
```

Comment:
Should this method be like `onFileChanged` or something like that?

Reply:
yeah but the FolderListener is in the same way, so I leave as it is to keep the consistence at least